src/vm/functions/special.rs

```diff
@@ -5,13 +5,14 @@ use vm::types::{Value, PrincipalData, QualifiedContractIdentifier};
 use vm::representations::{SymbolicExpression, SymbolicExpressionType};
 use vm::errors::Error;
 use vm::errors::{CheckErrors, InterpreterError, RuntimeErrorType, InterpreterResult as Result};
+use vm::contexts::GlobalContext;
 
 use chainstate::stacks::boot::boot_code_id;
 use chainstate::stacks::db::StacksChainState;
+use chainstate::stacks::events::{StacksTransactionEvent, STXEventType, STXLockEventData};
 use vm::clarity::ClarityTransactionConnection;
-use vm::database::ClarityDatabase;
 
-fn parse_pox_stacking_result(result: &Value) -> std::result::Result<(PrincipalData, u128, u128), i128> {
+fn parse_pox_stacking_result(result: &Value) -> std::result::Result<(PrincipalData, u128, u64), i128> {
```

Comment:
Why is this downgraded to a u64?

Reply:
u64 is our type of choice for representing block heights, if you want to use this output as an input, you'd be stuck with an oversized type that you'd have to cast again and again, when we should  really do it once when constructing this tuple.